domains/client/domain-operator/src/fraud_proof.rs

```diff
@@ -518,6 +518,9 @@ where
         mismatch_type: BundleMismatchType,
         bundle_index: u32,
         bad_receipt_hash: Block::Hash,
+        // Whether allow generate invalid proof against valid ER,
+        // only used in test
+        allow_invalid_proof: bool,
```

Comment:
Can we make this complied only during test please? 

Very easy to misuse or configure it wrong

Reply:
@teor2345 have done this in https://github.com/autonomys/subspace/pull/3369